Description: Use LDFLAGS from environment (dpkg-buildflags).
 Necessary for hardening flags.
.
 The buildsystem uses LFLAGS and LDFLAGS, but doesn't pass LDFLAGS when
 linking some binaries, fix that.
Author: Simon Ruderich <simon@ruderich.org>
Last-Update: 2012-09-14

Index: exim4-4.80/OS/Makefile-Linux
===================================================================
--- exim4-4.80.orig/OS/Makefile-Linux	2012-09-14 17:07:28.926489036 +0200
+++ exim4-4.80/OS/Makefile-Linux	2012-09-14 17:07:31.190489123 +0200
@@ -22,6 +22,8 @@
 XLFLAGS=-L$(X11)/lib
 X11_LD_LIB=$(X11)/lib
 
+LFLAGS=$(LDFLAGS)
+
 EXIWHAT_PS_ARG=ax
 EXIWHAT_EGREP_ARG='/exim4( |$$)'
 EXIWHAT_MULTIKILL_CMD=killall
Index: exim4-4.80/OS/Makefile-Base
===================================================================
--- exim4-4.80.orig/OS/Makefile-Base	2012-09-14 17:07:28.926489036 +0200
+++ exim4-4.80/OS/Makefile-Base	2012-09-14 17:07:31.190489123 +0200
@@ -113,7 +113,7 @@
 # Targets for special-purpose configuration header builders
 buildconfig: buildconfig.c
 	@echo "$(CC) buildconfig.c"
-	$(FE)$(CC) $(CFLAGS) $(INCLUDE) -o buildconfig buildconfig.c $(LIBS)
+	$(FE)$(CC) $(CFLAGS) $(LFLAGS) $(INCLUDE) -o buildconfig buildconfig.c $(LIBS)
 
 
 # Target for the exicyclog utility script
